napari/layers/labels/labels.py

```diff
@@ -243,7 +243,9 @@ class Labels(_ImageBase):
 
     brush_size_on_mouse_move = BrushSizeOnMouseMove(min_brush_size=1)
 
-    _move_modes: ClassVar[Dict[StringEnum, Callable[["Labels", Event], None]]] = {  # type: ignore[assignment]
+    _move_modes: ClassVar[
+        Dict[StringEnum, Callable[["Labels", Event], None]]
+    ] = {  # type: ignore[assignment]
```

Comment:
This is odd, might be a bug since pragme comments shouldn't matter: https://docs.astral.sh/ruff/formatter/black/#pragma-comments-are-ignored-when-computing-line-width